### ISSUE TITLE
[Regression] Window title is not set on Linux

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#63242]
### Describe the bug

Since some version months ago, unfortunately I did not bother to bisect and now its a tad too late, the window title is not set anymore.

### Steps to reproduce

1. Launch cataclysm-tiles (I have the non-tiles version not installed)
2. Notice `<unknown>` (my window managers fallback string) in the title bar of the spawned window

### Expected behavior

The window title should be set. Setting low priority because it is literally not affecting anything and I usually play in fullscreen anyways.

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: Arch; Description: Arch Linux; Release: rolling; Codename: n/a; 
- Game Version: 0.F-12810-gdf83f9a8413 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by Vaskritaya** [comment]
wasnt able to reproduce this using catapult launcher, the latest release from github, or quickly compiling my local repo. Using arch with i3wm.

**Comment by andrei8l** [comment]
Works fine here on mutter with both X11 and Wayland backends.

**Comment by NetSysFire** [comment]
Weird. I will need to compile a new version soon and see if its fixed.

**Comment by NetSysFire** [comment]
Can still reproduce on my machine. Any suggestions on how to debug this?

**Comment by andrei8l** [comment]
Title is set unconditionally once:
https://github.com/CleverRaven/Cataclysm-DDA/blob/18389fcce865aabd498336145779b3b143aef55d/src/main.cpp#L742

https://github.com/CleverRaven/Cataclysm-DDA/blob/18389fcce865aabd498336145779b3b143aef55d/src/translations.cpp#L101

so this is most likely an issue with SDL or the compositor.

**Comment by NetSysFire** [comment]
Hmm. I think the issue might be SDL. When I am at my desktop again, where I compile cdda, I will check what SDL version is used. I know I once built one for debugging and this one might still be used, that would be the cause.

**Comment by NetSysFire** [comment]
awesomewm. It worked in the past though and everything else works correctly.

**Comment by NetSysFire** [comment]
Found the issue: While debugging some more I discovered that my external sandboxing via firejail is the cause. While this sounds very much like user error, it worked in the past and something changed somewhere. Either here or in SDL. I assume it wants something in /etc to be visible, so I will have to figure out why it fails that way without any error and why it does not set the title at all. I'll close this for now until I know for sure this is not a SDL bug.
